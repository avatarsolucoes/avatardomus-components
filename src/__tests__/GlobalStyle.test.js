// import React from 'react';
// import renderer from 'react-test-renderer';

import initialState from '../GlobalStyle/globalTheme';

describe('GlobalStyle', () => {
  it('is truthy initialState', () => {
    expect(initialState).toBeTruthy();
    const state = JSON.stringify(initialState);
    expect(state).toMatchSnapshot();
  });
});
