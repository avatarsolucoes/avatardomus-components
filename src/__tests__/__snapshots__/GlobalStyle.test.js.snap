// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalStyle is truthy initialState 1`] = `"{\\"primary\\":\\"#0E2841\\",\\"secundary\\":\\"#0D7ECC\\",\\"contrast\\":\\"#9C1B3A\\",\\"bgprimary\\":{\\"color\\":\\"#FFF\\",\\"background\\":\\"#0E2841\\"},\\"bgsecundary\\":{\\"color\\":\\"#FFF\\",\\"background\\":\\"#0D7ECC\\"},\\"bgcontrast\\":{\\"color\\":\\"#FFF\\",\\"background\\":\\"#9C1B3A\\"}}"`;
